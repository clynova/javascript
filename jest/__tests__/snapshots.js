const cliente = {
    nombre: "Pedro Sanchez",
    balance: 500,
    tipo: "Premium"
}


describe('Testing al Cliente', () => {
    test('Es Pedro Sanchez', () => {
        expect(cliente).toMatchSnapshot('Pedro Sanchez')
    });
    
});
