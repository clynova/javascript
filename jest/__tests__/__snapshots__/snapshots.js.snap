// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al Cliente Es Pedro Sanchez: Pedro Sanchez 1`] = `
{
  "balance": 500,
  "nombre": "Pedro Sanchez",
  "tipo": "Premium",
}
`;
